emba.sh

```diff
@@ -397,6 +496,7 @@ main()
       l)
         export LOG_DIR="$OPTARG"
         export TMP_DIR="$LOG_DIR""/tmp"
+        export CSV_DIR="$LOG_DIR""/csv_logs"
```

Comment:
Line 383 is the default value. If this is changed via an option in the container we need to adjust this again for the container. Otherwise it is going to setup in a wrong way.

Reply:
Thanks for clarifying it.